### PR TITLE
Added placement example for dynamic parts

### PR DESCRIPTION
**Pull request by brunoAltinet**

### CODE REVIEW

File: src/docs/reference/core/Placement/README.md [hunk 1]
```diff
@@ -319,6 +319,60 @@ We also specify that the `Content` column will take 9 columns, of the default 12
...
+
+Alternatively, if you find this complicated for dynamic parts, you can display your dynamic part like so:
+
+=== Content-Product.Summary.html
+``` html
+@await DisplayAsync(Model.Content.GalleryPart)
+```
+
+!!! note
+    Directly displaying parts like shown above without placement is a bit hacky. If for example we would call both `@await DisplayAsync(Model.Content.GalleryPart)` and `@await DisplayAsync(Model.Content)` inside the same template, we would expect both calls to show gallery. However, only the first call will show the gallery, and the other would ignore the gallery shape (since it's already rendered).
```
Comment (ns8482e) [inline src/docs/reference/core/Placement/README.md:375]: IMHO  - In documentation, objective is to clear the doubt - we should avoid wording that creates doubts in readers mind.

Comment (brunoAltinet) [reply src/docs/reference/core/Placement/README.md:375]: Yeah, i'm not completely for keeping this in. Although i like the fact you can do that.

**Review by ns8482e — APPROVED** [review]

File: src/docs/reference/core/Placement/README.md [hunk 2]
```diff
@@ -319,6 +319,50 @@ We also specify that the `Content` column will take 9 columns, of the default 12
...
+  "ContentPart": [
+    {
+      "place": "MyGalleryZone",
+      "differentiator": "GalleryPart"
+    }
+  ]
+}
+```
+
+This setup would then show your template (e.g. `GalleryPart.cshtml` or `GalleryPart.Detail.cshtml`) where DisplayAsync was called.
```
Comment (hishamco) [inline src/docs/reference/core/Placement/README.md:344]: `DisplayAsync`

File: src/docs/reference/core/Placement/README.md [hunk 3]
```diff
@@ -319,6 +319,50 @@ We also specify that the `Content` column will take 9 columns, of the default 12
...
+
+If we would like to show the same part in a summary display content template (or any other that isn't `Detail` display type):
+
```
Comment (hishamco) [inline src/docs/reference/core/Placement/README.md:347]: Remove extra line

File: src/docs/reference/core/Placement/README.md [hunk 4]
```diff
@@ -319,6 +319,50 @@ We also specify that the `Content` column will take 9 columns, of the default 12
...
+{
+  "ContentPart_Summary": [
+    {
+      "place": "MyGalleryZone",
+      "differentiator": "GalleryPart"
+    }
+  ]
+}
+```
+This setup would then show your template  (e.g. `GalleryPart.cshtml` or `GalleryPart.Summary.cshtml`) where DisplayAsync was called.
```
Comment (hishamco) [inline src/docs/reference/core/Placement/README.md:365]: Add line before and after code snippet

**Review by hishamco — CHANGES_REQUESTED** [review]
LGTM after fixing the format

File: src/docs/reference/core/Placement/README.md (hunk 4 above)
Comment (hishamco) [reply src/docs/reference/core/Placement/README.md:366]: `DisplayAsync`

**Review by hishamco — CHANGES_REQUESTED** [review]

**Review by hishamco — APPROVED** [review]

**Pull request merged**